frontend/src/modules/machinery/components/TranslationMemory.js

```diff
@@ -25,9 +26,13 @@ export default function TranslationMemory(props: Props) {
                     rel='noopener noreferrer'
                     onClick={(e: SyntheticMouseEvent<>) => e.stopPropagation()}
                 >
-                    <Localized id='machinery-TranslationMemory--translation-memory'>
-                        <span>TRANSLATION MEMORY</span>
-                    </Localized>
+                    {!props.projectName ? (
+                        <Localized id='machinery-TranslationMemory--translation-memory'>
+                            <span>TRANSLATION MEMORY</span>
+                        </Localized>
+                    ) : (
+                        <span>{props.projectName.toUpperCase()}</span>
```

Comment:
I think linking to the translate view would be very nice. Once/if @jotes is able to add those return values, I'll make the change.

Reply:
It turned out this feature is blocked by https://bugzilla.mozilla.org/show_bug.cgi?id=1324933 (same as https://bugzilla.mozilla.org/show_bug.cgi?id=1390931), because we're grouping multiple TM entries together and we'd need to link to all of them. So we decided to drop the effort within this patch.

Sorry for the confusion!